### PR TITLE
Resolve pagination for pre-validation scan

### PR DESCRIPTION
**Pull request by PoulainMaxime**

### CODE REVIEW

File: _dev/src/components/product-feed-page/product-feed-pre-scan-table-status-details.vue [hunk 1]
```diff
@@ -66,107 +66,125 @@
...
+              :aria-sort="type.sortable ? 'none' : null"
+            >
+              {{ type.label }}
+            </b-th>
+          </b-tr>
+        </b-thead>
+        <b-tbody>
+          <template v-if="apiError">
+            <tr>
+              <td class="text-center" :colspan="fields.length">
```
Comment (elisegriset92) [inline _dev/src/components/product-feed-page/product-feed-pre-scan-table-status-details.vue:92]: Require linter

File: _dev/src/store/modules/product-feed/mutations.ts [hunk 2]
```diff
@@ -170,7 +170,7 @@ export default {
   [MutationsTypes.SET_PRESCAN_NEXT_PAGE](state: LocalState, payload: number) {
     state.preScanDetail.currentPage = payload;
   },
-  [MutationsTypes.SET_PRESCAN_PRODUCTS](state: LocalState, payload: PreScanReporting[]) {
+  [MutationsTypes.SET_PRESCAN_PRODUCTS](state: LocalState, payload) {
```
Comment (Quetzacoalt91) [inline _dev/src/store/modules/product-feed/mutations.ts:173]: Why did you remove the type?

File: _dev/src/components/product-feed-page/product-feed-pre-scan-table-status-details.vue [hunk 3]
```diff
@@ -66,107 +66,125 @@
...
-        </template>
-
-        <template #cell(price)="data">
-          <span
-            class="material-icons"
-            :class="data.item.isMissingPrice ? 'text-danger' : 'text-success'"
+            <b-tr :key="index">
+              <b-td class="align-top">
+                {{ product.productId }}
+                {{ product.attributeId > 0 ? '&#8209;' + product.attributeId : '' }}
```
Comment (Quetzacoalt91) [inline _dev/src/components/product-feed-page/product-feed-pre-scan-table-status-details.vue:111]: Maybe add a space between the hyphen and the attribute id ?

**Comment by Quetzacoalt91** [comment]
Can you rename this PR so the merchant understands what has been changed for him?

**Review by Quetzacoalt91 — APPROVED** [review]
